src/test/features/staging-spec.ts

```diff
@@ -114,6 +114,20 @@ suite('ModelViewerElementBase with StagingMixin', () => {
                                // rAF though
         expect(element.turntableRotation).to.be.greaterThan(turntableRotation);
       });
+
+      suite('when the model is not visible', () => {
+        setup(() => {
+          Object.defineProperty(
+            element, 'modelIsVisible', {value:false});
```

Comment:
Is this how we're mocking getters in this repo?

Reply:
This is a totally reasonable way to do it, ya! 👌